code/modules/mob/language/alien/049.dm

```diff
@@ -0,0 +1,9 @@
+/datum/language/scp049
+	name = LANGUAGE_SCP049_GLOBAL
+	desc = "SCP 049 has the ability to telepathically communicate with his creations."
```

Comment:
This should be handled like cult communion, not as a language thats fucking wack

Reply:
Isn't cult communion also a language? Having to use a verb to speak seemed very clunky to me.

Reply:
Handled via an action is what Im trying to say

Reply:
Languages have hotkeys like ,g to use them quickly. I don't mind using a verb/action for communication, but it seems more convenient to talk this way.

Reply:
I guess that works, might want to add it to their text on-spawn